.gitignore

```diff
@@ -1,4 +1,6 @@
+dist/*
```

Comment:
unfortunately `dist` is required folder for submit grafana plugin to grafana-plugins-repository